code/game/objects/items/weapons/implant/implants/carrion/breeding_spider.dm

```diff
@@ -14,7 +14,7 @@
 	if(!wearer)
 		to_chat(owner_mob, SPAN_WARNING("[src] doesn't have a host"))
 		return
-	if(!ishuman(wearer))
+	if(!(wearer.type == /mob/living/carbon/human))
```

Comment:
You can write as if(!ishuman(wearer.type))

The code below this line shows it can.